package.json

```diff
@@ -11,16 +11,20 @@
   },
   "dependencies": {
     "@reduxjs/toolkit": "^1.9.2",
+    "grommet": "^2.28.0",
+    "lodash": "^4.17.21",
     "mapbox-gl": "^2.12.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-map-gl": "^7.0.21",
     "react-redux": "^8.0.5",
     "react-router": "^6.8.1",
     "react-router-dom": "^6.8.1",
-    "redux": "^4.2.1"
+    "redux": "^4.2.1",
+    "styled-components": "^5.3.9"
```

Comment:
note: had to install because grommet was barfing on more recent styled-components versions